File: example/javalib/testing/5-test-stealing/foo/test/src/foo/RandomTestsA.java
package foo;

import org.junit.Test;

public class RandomTestsA extends RandomTestsUtils {
    @Test
    public void test1() throws Exception { testGreeting("Storm", 38); }
    @Test
    public void test2() throws Exception { testGreeting("Bella", 25); }
    @Test
    public void test3() throws Exception { testGreeting("Cameron", 32); }
} 

File: example/kotlinlib/testing/5-test-stealing/foo/test/src/foo/RandomTestsA.kt
package foo

import org.junit.jupiter.api.Test

class RandomTestsA : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() {
        testGreeting("Storm", 38)
    }

    @Test
    @Throws(Exception::class)
    fun test2() {
        testGreeting("Bella", 25)
    }

    @Test
    @Throws(Exception::class)
    fun test3() {
        testGreeting("Cameron", 32)
    }
}
